$MirOS: ports/net/curl/patches/patch-tests_server_sws_c,v 1.1 2008/04/05 21:54:40 tg Exp $
--- tests/server/sws.c.orig	Thu Apr 24 09:42:28 2008
+++ tests/server/sws.c	Tue Aug 26 19:03:21 2008
@@ -673,7 +673,7 @@ static int send_doc(curl_socket_t sock, 
       logmsg("Identifying ourselves as friends");
       sprintf(msgbuf, "WE ROOLZ: %ld\r\n", (long)getpid());
       msglen = strlen(msgbuf);
-      sprintf(weare, "HTTP/1.1 200 OK\r\nContent-Length: %d\r\n\r\n%s",
+      sprintf(weare, "HTTP/1.1 200 OK\r\nContent-Length: %zd\r\n\r\n%s",
               msglen, msgbuf);
       buffer = weare;
       break;
